packages/web/package.json

```diff
@@ -46,19 +49,42 @@
     "@radix-ui/react-toggle": "^1.1.0",
     "@radix-ui/react-tooltip": "^1.1.4",
     "@replit/codemirror-lang-csharp": "^6.2.0",
+    "@replit/codemirror-lang-nix": "^6.0.1",
+    "@replit/codemirror-lang-solidity": "^6.0.2",
+    "@replit/codemirror-lang-svelte": "^6.0.0",
     "@replit/codemirror-vim": "^6.2.1",
+    "@shopify/lang-jsonc": "^1.0.0",
+    "@ssddanbrown/codemirror-lang-twig": "^1.0.0",
     "@tanstack/react-query": "^5.53.3",
     "@tanstack/react-table": "^8.20.5",
     "@tanstack/react-virtual": "^3.10.8",
     "@uiw/codemirror-themes": "^4.23.6",
     "@uiw/react-codemirror": "^4.23.0",
+    "@viz-js/lang-dot": "^1.0.4",
+    "@xiechao/codemirror-lang-handlebars": "^1.0.4",
     "class-variance-authority": "^0.7.0",
     "client-only": "^0.0.1",
     "clsx": "^2.1.1",
+    "cm6-graphql": "^0.2.0",
+    "codemirror": "^5.65.3",
+    "codemirror-lang-brainfuck": "^0.1.0",
+    "codemirror-lang-elixir": "^4.0.0",
+    "codemirror-lang-hcl": "^0.0.0-beta.2",
+    "codemirror-lang-j": "^0.1.1",
+    "codemirror-lang-jq": "^1.0.0",
+    "codemirror-lang-makefile": "^0.1.1",
+    "codemirror-lang-mermaid": "^0.5.0",
+    "codemirror-lang-perl": "^0.1.5-beta.6",
+    "codemirror-lang-prolog": "^0.1.0",
+    "codemirror-lang-r": "^0.1.1",
+    "codemirror-lang-sparql": "^2.0.0",
+    "codemirror-lang-spreadsheet": "^1.3.0",
+    "codemirror-lang-zig": "^0.1.0",
     "embla-carousel-auto-scroll": "^8.3.0",
     "embla-carousel-react": "^8.3.0",
     "escape-string-regexp": "^5.0.0",
     "fuse.js": "^7.0.0",
+    "graphql": "^16.9.0",
```

Comment:
do we need to take a dependency on graphql?

Reply:
It's a peer dependency for qraphql syntax highlighting.